### ISSUE TITLE
[Feature request]: shuffle playlist

### ISSUE THREAD

**Issue by saifg12345** [pystardust/ytfzf#732]
Is there any way in ytfzf to add shuffling features to playlists? I was wondering but I don't really have any bash scripting knowledge as far as scripting goes.

**Comment by Euro20179** [comment]
Depends on your video player, ytfzf uses mpv by default which i think does have playlist shuffling

> --shuffle
Play files in random order.

You could just select all the items in a playlist (or the playlist itself) and give it to mpv, and have `--url-handler-opts=--shuffle` and mpv will shuffle the playlist.

**Comment by saifg12345** [comment]
I tried this and it didn't work. Do I have to manually set it in mpv?

**Comment by Euro20179** [comment]
I think you need `--type==playlist`,

but if that doesn't work then i'm not sure.

> Do I have to manually set it in mpv

No harm in trying.

**Comment by saifg12345** [comment]
Yeah, I just ended up playing my music with mpv only. Thanks for the help though!

**Comment by tabletseeker** [comment]
Updated and working ytfzf project here:
https://github.com/tabletseeker/ytfzf_prime
